src/sentry/static/sentry/app/components/charts/worldMapChart.jsx

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import BaseChart from './baseChart';
+import MapSeries from './series/mapSeries';
+
+export default class WorldMapChart extends React.Component {
+  static propTypes = {
+    ...BaseChart.propTypes,
+    seriesOptions: PropTypes.object,
+  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      countryCodesMap: null,
+    };
+  }
+
+  async componentDidMount() {
+    const countryCodesMap = await import(/* webpackChunkName: "countryCodesMap" */ 'app/data/countryCodesMap');
+
+    // eslint-disable-next-line
+    this.setState({
+      countryCodesMap: countryCodesMap.default,
+    });
+  }
+
+  render() {
+    if (this.state.countryCodesMap === null) {
+      return null;
```

Comment:
Loading indicator?

Reply:
Going to hold off on this for now as requirements may change for this when we start to use it.